docs/control-structures.rst

```diff
@@ -409,15 +409,22 @@ In any case, you will get a warning about the outer variable being shadowed.
 Error handling: Assert, Require, Revert and Exceptions
 ======================================================
 
-Solidity uses state-reverting exceptions to handle errors. Such an exception undoes all changes made to the
-state in the current call (and all its sub-calls) and flags an error to the caller.
-
-When exceptions happen in a sub-call, they "bubble up" (i.e., exceptions are rethrown) automatically. Exceptions to this rule are ``send``
-and the low-level functions ``call``, ``delegatecall`` and ``staticcall``, they return ``false`` as their first return value in case
+Solidity uses state-reverting exceptions to handle errors.
+Such an exception undoes all changes made to the
+state in the current call (and all its sub-calls) and
+flags an error to the caller.
+
+When exceptions happen in a sub-call, they "bubble up" (i.e.,
+exceptions are rethrown) automatically. Exceptions to this rule are ``send``
+and the low-level functions ``call``, ``delegatecall`` and
+``staticcall``, they return ``false`` as their first return value in case
```

Comment:
```suggestion
``staticcall`` that return ``false`` as their first return value in case
```

Reply:
What about ` - they return...`? This "they" should also include `send`.

Reply:
okay, perhaps a colon then. Like 

```
Exceptions to this rule are ``send``
￼and the low-level functions ``call``, ``delegatecall`` and
￼``staticcall``: they return ``false`` as their first return value
```

Reply:
Sounds good!